var/spack/repos/builtin/packages/osu-micro-benchmarks/package.py

```diff
@@ -44,3 +45,21 @@ def configure_args(self):
             config_args.append('LDFLAGS=-lrt')
 
         return config_args
+
+    def setup_environment(self, spack_env, run_env):
+        run_env.prepend_path(
```

Comment:
If you want to save space, you can create a variable:
```python
mpidir = join_path(self.prefix.libexec, 'osu-micro-benchmarks', 'mpi')
```
and then append the single subdir needed for each directory.

Reply:
thanks, good rec, i'll pursue this